### ISSUE TITLE
Umbrella header for module 'TwitterCore' does not include header 'TWTRColorUtil.h'

### ISSUE THREAD

**Issue by dmcgloin** [twitter/twitter-kit-ios#77]
When building a Swift app that includes latest via CocoaPods:  Twitter Kit 3.4.0 and Twitter Core 3.2.0, I get the following warning (which is new):

`/Users/<me>/Documents/GitHub/<myapp>/<module-includes>:1:1: Umbrella header for module 'TwitterCore' does not include header 'TWTRColorUtil.h'`

**Comment by shroff-kandarp** [comment]
same issue here

**Comment by Quanda** [comment]
Any resolution to this? Getting same error

**Comment by dmcgloin** [comment]
Feels like the time has come to remove Twitter support - at least from our app.

**Comment by AlexGingell** [comment]
Has anyone managed to obtain a version of TwitterKit v3.4.0 that is error free?  I have checked many forks and branches and cannot see any error-free versions which compile error-free and produce no issues.  If it is not this issue it is 
> Private submodule 'TwitterCore.Private' in private module map, expected top-level module
> Rename 'TwitterCore.Private' to ensure it can be found by name

**Comment by npu3pak** [comment]
Got some working solution. It fixed all TwitterCore related warnings for me. 
https://gist.github.com/npu3pak/108d8332a774289c08472d6fd85a1bbb

**Comment by oliverbytes** [comment]
I'm getting the same issue only on a specific mac computer. on my laptop it's working fine. Any final solution? @npu3pak suggestion didn't work for me

**Comment by npu3pak** [comment]
> I'm getting the same issue only on a specific mac computer. on my laptop it's working fine. Any final solution? @npu3pak suggestion didn't work for me

You can try to fix module.private.modulemap and TwitterCore.h manually and then use some script on build phase to automate the process.

**Comment by RobertChalmers** [comment]
Yes, npu3pak's fix for ColorUtil works nice. Now if I can just fix the rest of the depreciated problems is the SBSDKSharKit it'll be nice.

I'm wondering if anyone - the original repo builder? is even still working on this?
